orion/pkg/domains/all_domains.go

```diff
@@ -0,0 +1,10 @@
+package domains
+
+type Domains struct {
```

Comment:
Rename to `UnpublishedlDomains` because announcements and users are not part of this struct